src/main/java/com/bookstore/entity/User.java

```diff
@@ -0,0 +1,35 @@
+package com.bookstore.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Setter
+@Getter
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false, unique = true)
+    private String email;
+
+    @Column(nullable = false)
+    private String password;
+
+    @Column(nullable = false, name = "first_name")
+    private String firstName;
+
+    @Column(nullable = false, name = "last_name")
+    private String lastName;
+
+    @Column(name = "shipping_address")
```

Comment:
Don't need to add `name` attribute to Column annotation. Hibernate automatically creates column name in snake_case format

```suggestion
    @Column(nullable = false)
    private String firstName;
    @Column(nullable = false)
    private String lastName;
```